GrappleGirlGame/main.lua

```diff
@@ -1,66 +1,100 @@
 require("Character")
 require("love")
-FLOOR_CATAGORY = 1
+FLOOR_CATEGORY = 4
```

Comment:
You should probably move all these category constants to a `constants.lua` file or something similar like that. You could split it into different tables, like CATEGORIES.FLOOR, for example, and group other similar constants together as well.